styles/components/footer.css

```diff
@@ -5,6 +5,13 @@
   bottom: 0;
 }
 
+.footer__post-view {
+  display: felx;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+}
+
 .navigation {
```

Comment:
The file name "footer" does not fit to the class names. One file per component - both must have the same name.